### PR TITLE
Add Python 3 compatibility.

### PR DESCRIPTION
**Pull request by somexpert**
This also drops the tests and formal support for Python 2.6, although it should continue to work with that version for the foreseeable future.

### CODE REVIEW

**Comment by somexpert** [comment]
ping @ldko

File: pypairtree/pairtreelist.py [hunk 1]
```diff
@@ -33,9 +33,9 @@ def listIDs(basedir):
         for obj in objects:
             doi = os.path.split(obj)[1]
             # print with prefix and original chars in place
-            print prefix + pairtree.deSanitizeString(doi)
```
Comment (ldko) [inline pypairtree/pairtreelist.py:36]: What do you think about adding the `from __future__ import print_function` import to this file?

Comment (somexpert) [reply pypairtree/pairtreelist.py:36]: Don't mind adding it. Doesn't hurt anything.

**Review by ldko — COMMENTED** [review]
What was the dropping of 3.7 support about?

**Comment by somexpert** [comment]
Doesn't look like Travis can build an env with Python 3.7 yet. I think I could use a dev branch of Python 3.7, but I figured I'd just wait to add the env until it was available fully. [See here.](https://docs.travis-ci.com/user/languages/python/#development-releases-support)

**Comment by ldko** [comment]
ok. :+1:

**Pull request merged**